packages/x-codemod/README.md

```diff
@@ -119,7 +139,26 @@ npx @mui/x-codemod v7.0.0/data-grid/preset-safe <path|folder>
 
 The list includes these transformers
 
-NO CODEMOD ADDED YET
+- [`rename-components-to-slots-data-grid`](#rename-components-to-slots-data-grid)
+
+#### `rename-components-to-slots-data-grid`
+
+Renames the `components` and `componentsProps` props to `slots` and `slotProps`, respectively.
```

Comment:
```suggestion
Renames the `components` and `componentsProps` props to `slots` and `slotProps` respectively.
```

Reply:
https://www.internationalscienceediting.com/how-to-use-respectively-correctly/#:~:text=If%20at%20the%20end%20of,is%20placed%20between%20two%20commas. :angel:

Reply:
The more you know. 🧠 🤯 
Seems like WebStorm really cares about correct grammer. 😮

Reply:
Sometimes it's totally wrong though :laughing: